website/content/en/docs/config/environment-variables.md

```diff
@@ -139,6 +139,14 @@ This page documents the environment variables used in Lima.
   ```sh
   export LIMA_USERNET_RESOLVE_IP_ADDRESS_TIMEOUT=5
   ```
+### `LIMA_VZ_ASIF`
```

Comment:
Can we implement this as a vmOpt rather than an env var?

Reply:
Changed to `vmOpts.vz.diskImageFormat`